tsup.config.ts

```diff
@@ -25,4 +25,11 @@ export default defineConfig([
     entry: {'api-client': 'src/index.ts'},
     shims: true, // replace 'import.meta' references
   },
+  {
+    ...commonConfig,
+    format: 'iife',
```

Comment:
and i have no idea why we need `iife` and if it's best solution - this is just "first" way to convince `tsup` to _bundle_ all required deps into one file

note: `build/worker.js` contains normal `import` clauses so it works only in context where `imports` to work ... and they don't work in scripts loaded from `blob:` and/or in sandboxed iframes

Reply:
Huh! Interesting – I'm OK with going this direction for now, if we find another way to bundle the imports later we can always change it back. 

Not necessary in this PR, but later we could add a short unit test that checks to make sure there are no imports in the `./worker-compat` script.